arkouda/message.py

```diff
@@ -97,6 +99,9 @@ def __init__(
             The request message format
         args : str
             The delimited string containing the command arguments
+        size : str
```

Comment:
should this be in int?

Reply:
I had everything set to string because there were issues when I was using int initially. Maybe I set something up wrong, I will try that again.

Reply:
Apparently I had messed something up. Will be pushing up with `int` shortly.

Reply:
@Ethan-DeBandi99 sounds cool!

Reply:
looks good